cpp/pipelines/add_subscription_bundle_entries.cc

```diff
@@ -37,18 +37,22 @@ using BundleToPPNsMap = std::map<std::string, std::set<std::string>>;
 
 
 [[noreturn]] void Usage() {
- std::cerr << "Usage: " << ::progname << "marc_input marc_output\n"
+ std::cerr << "Usage: " << ::progname << " marc_input marc_output\n"
            << "Generate a dummy entry for subscriptions from the configuration given in journal_alert_bundles.conf\n";
  std::exit(EXIT_FAILURE);
 }
 
 
-MARC::Record GenerateBundleRecord(const std::string &record_id, const std::string &bundle_name, const std::vector<std::string> &instances) {
+MARC::Record GenerateBundleRecord(const std::string &record_id, const std::string &bundle_name, const std::vector<std::string> &instances,
+                                  const std::string &description)
+{
     const std::string today(TimeUtil::GetCurrentDateAndTime("%y%m%d"));
     // exclude from Ixtheo e.g. because it's a pure Relbib list
     const bool exclude_ixtheo(std::find(instances.begin(), instances.end(), "ixtheo") == instances.end());
     const bool include_relbib(std::find(instances.begin(), instances.end(), "relbib") != instances.end());
-    const bool include_bibstudies(std::find(instances.begin(), instances.end(), "bistudies") != instances.end() ? true : false);
+    const bool include_bibstudies(std::find(instances.begin(), instances.end(), "bibstudies") != instances.end() ? true : false);
```

Comment:
Please get rid off the part starting with the question mark.  Also on the next statement.